website/content/en/preview/concepts/nodeclasses.md

```diff
@@ -1157,15 +1191,40 @@ status:
 
 ## status.amis
 
-[`status.amis`]({{< ref "#statusamis" >}}) contains the resolved `id`, `name`, and `requirements` of either the default AMIs for the [`spec.amiFamily`]({{< ref "#specamifamily" >}}) or the AMIs selected by the [`spec.amiSelectorTerms`]({{< ref "#specamiselectorterms" >}}) if this field is specified.
+[`status.amis`]({{< ref "#statusamis" >}}) contains the resolved `id`, `name`, and `requirements` of the AMIs selected by the [`spec.amiSelectorTerms`]({{< ref "#specamiselectorterms" >}}).
 
-#### Examples
+When discovering AMIs via id, owner, tags, and/or name, Karpenter will only discover a `kubernetes.io/arch` requirement.
+
+```yaml
+spec:
+  amiSelectorTerms:
+    - tags:
+        karpenter.sh/discovery: "${CLUSTER_NAME}"
+status:
+  amis:
+  - id: ami-01234567890123456
+    name: custom-ami-amd64
+    requirements:
+    - key: kubernetes.io/arch
+      operator: In
+      values:
+      - amd64
+  - id: ami-01234567890123456
+    name: custom-ami-arm64
+    requirements:
+    - key: kubernetes.io/arch
+      operator: In
+      values:
+      - arm64
+```
 
-Default AMIs resolved from the AL2 AMIFamily:
+However, specifying an `eksOptimized` term may allow Karpenter to discover more requirements.
```

Comment:
May allow? Seems like it does allow.

Reply:
It depends on the AMI family, which is why I opted for "may allow". For example, AL2023 only discovers architecture which we would be able to discover using other AMI selector terms.

Reply:
Well, Karpenter is always allowed to discover more requirements. It just doesn't if they don't exist. I'm arguing semantics, but I'd generally avoid words like "may" since it make things less clear to people. It leads to the question, "Does it or doesn't it?"

Reply:
Good point, I'll see what I can come up with to reword. Basically just want to make sure it's clear that not all `eksOptimized` families will be able to discover GPU AMIs and such.